src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -0,0 +1,2394 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents an expression that creates a pointer value by taking the address of a reference.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    internal sealed partial class AddressOfExpression : Operation, IAddressOfExpression
```

Comment:
Intresting design question: as these are now a full new tree, is there value in having the interfaces at all?  Let's discuss this at our meeting.  The interface approach was necessayr so that we could implement them on the bound-nodes across VB/C#.  But if bound-nodes are not the objects anymore, we don't really need interfaces.